File: GCD_LCM_Calculator.cs
```CSharp
using System;

class Program
{
    static void Main(string[] args)
    {
        int num1, num2, gcd, lcm;
        Console.Write("Enter the First Number : ");
        num1 = int.Parse(Console.ReadLine());
        Console.Write("Enter the Second Number : ");
        num2 = int.Parse(Console.ReadLine());
        gcd = GCD(num1, num2);
        lcm = LCM(num1, num2);
        Console.WriteLine("GCD of {0} and {1} is {2}", num1, num2, gcd);
        Console.WriteLine("LCM of {0} and {1} is {2}", num1, num2, lcm);
        Console.ReadLine();
    }

    static int GCD(int a, int b)
    {
        if (b == 0)
            return a;
        return GCD(b, a % b);
    }

    static int LCM(int a, int b)
    {
        return (a * b) / GCD(a, b);
    }
}
```

File: GCD_LCM_Calculator.py
def gcd(a,b):
    if b==0:
        return a
    else:
        return gcd(b,a%b)

def lcm(a,b):
    return a*b//gcd(a,b)

while True:
    print("1. GCD")
    print("2. LCM")
    print("3. Exit")
    choice = int(input("Enter your choice: "))
    if choice == 1:
        num1 = int(input("Enter first number: "))
        num2 = int(input("Enter second number: "))
        print("GCD of", num1, "and", num2, "is", gcd(num1, num2))
    elif choice == 2:
        num1 = int(input("Enter first number: "))
        num2 = int(input("Enter second number: "))
        print("LCM of", num1, "and", num2, "is", lcm(num1, num2))
    elif choice == 3:
        break
    else:
        print("Invalid choice!!")